### PR TITLE
doc: Update the EKS getting started guide

### PR DESCRIPTION
**Pull request by michi-covalent**
- Create a cluster with `--without-nodegroup` option and create a node
   group after deploying Cilium.
- Change the cluster name to `test-cluster` in the ENI getting started
   so it can use the same "Create a nodegroup" section as the EKS guide.

Ref: https://github.com/weaveworks/eksctl/pull/2022

Signed-off-by: Michi Mutsuzaki <michi@isovalent.com>

### CODE REVIEW

**Comment by errordeveloper** [comment]
Hey @michi-covalent, just a couple of nits!

1. Would you mind to use either long or short names for all flag please?

So either:
```
eksctl create cluster -n test-cluster -N 0 -M 2
```
Or:
```
eksctl create cluster --name test-cluster --nodes 0 --nodes-max 2
```
If it's best if you could make it consistent of the entire guide too.

2. Perhaps we could explicitly create the nodegroup in separate step instead of creating one with 0 nodes and scaling up later. So overall the flow would be like this:

```
eksctl create cluster --name test-cluster --without-nodegroup
kubectl delet --namespace kube-system  daemonset/aws-node
helm install ...
eksctl create nodegroup --cluster test-cluster
```

What do you think?

**Comment by michi-covalent** [comment]
@errordeveloper yeah i like the second option to create a cluster without nodegroup first. let me try what you suggested, and i'll update this pr accordingly.

**Comment by michi-covalent** [comment]
i'm seeing warnings like these after creating a nodegroup. not sure if i got the same error before. trying scaling the cluster again to check.
```
level=info msg="Retrieved node information from kubernetes node" nodeName=ip-192-168-50-63.ec2.internal subsys=k8s
level=warning msg="Waiting for k8s node information" error="required IPv4 pod CIDR not present in node resource" subsys=k8s
level=info msg="Retrieved node information from kubernetes node" nodeName=ip-192-168-50-63.ec2.internal subsys=k8s
level=warning msg="Waiting for k8s node information" error="required IPv4 pod CIDR not present in node resource" subsys=k8s
level=info msg="Retrieved node information from kubernetes node" nodeName=ip-192-168-50-63.ec2.internal subsys=k8s
level=warning msg="Waiting for k8s node information" error="required IPv4 pod CIDR not present in node resource" subsys=k8s
level=info msg="Retrieved node information from kubernetes node" nodeName=ip-192-168-50-63.ec2.internal subsys=k8s
level=warning msg="Waiting for k8s node information" error="required IPv4 pod CIDR not present in node resource" subsys=k8s
```

**Comment by michi-covalent** [comment]
discussion on slack: https://app.slack.com/client/T1MATJ4SZ/C1MATJ5U5

**Comment by aanm** [comment]
@michi-covalent I've marked the PR as draft since it looked like there's still some work being done

**Comment by michi-covalent** [comment]
ready for review cc @errordeveloper

**Comment by michi-covalent** [comment]
test-me-please

**Review by errordeveloper — APPROVED** [review]
Overall, LGTM! Few nits, if you have a moment :)

File: Documentation/gettingstarted/k8s-install-eks.rst [hunk 1]
```diff
@@ -59,7 +59,7 @@ details on how to set credentials, change region, VPC, cluster size, etc.
 
    .. code:: bash
 
-     eksctl create cluster -n test-cluster -N 0
+     eksctl create cluster -n test-cluster --without-nodegroup
```
Comment (errordeveloper) [inline Documentation/gettingstarted/k8s-install-eks.rst]:
```suggestion
     eksctl create cluster --name test-cluster --without-nodegroup
```
Just a nit...

File: Documentation/gettingstarted/aws-eni.rst [hunk 2]
```diff
@@ -25,7 +25,7 @@ EKS cluster using ``eksctl``, see the section :ref:`k8s_install_eks`.
 
 .. code:: bash
 
-   eksctl create cluster -n eni-cluster -N 0
+   eksctl create cluster -n test-cluster --without-nodegroup
```
Comment (errordeveloper) [inline Documentation/gettingstarted/aws-eni.rst]:
```suggestion
   eksctl create cluster --name test-cluster --without-nodegroup
```

File: Documentation/gettingstarted/k8s-install-eks.rst (hunk 1 above)
Comment (michi-covalent) [reply Documentation/gettingstarted/k8s-install-eks.rst]: done

File: Documentation/gettingstarted/aws-eni.rst (hunk 2 above)
Comment (michi-covalent) [reply Documentation/gettingstarted/aws-eni.rst]: done

**Pull request merged**